.circleci/config.yml

```diff
@@ -187,6 +191,9 @@ workflows:
           requires:
             - install
       - test-unit:
+          matrix:
+            parameters:
+              react-major-version: ['16', '17', '18']
```

Comment:
This is to run our unit tests three times with a different value in the `REACT_MAJOR_VERSION` environment variable.